test/powershell/engine/Api/TypeInference.Tests.ps1

```diff
@@ -926,6 +926,33 @@ Describe "Type inference Tests" -tags "CI" {
         $res.Name | Should -Be System.Int32
     }
 
+    It 'Infers type of variable $_ in catch block' {
+        $variableAst = { try {} catch { $_ } }.Ast.Find({ param($a) $a -is [System.Management.Automation.Language.VariableExpressionAst] }, $true)
+        $res = [AstTypeInference]::InferTypeOf($variableAst)
+
+        $res.Count | Should -Be 1
```

Comment:
`$res | Should -HaveCount 1`

Reply:
Could you please address the comment?
Line 941 too.

Reply:
All done. There are a few tests for other things that should probably use this as well, but I think that might be better left to another PR specifically for test revisions.